### ISSUE TITLE
SMF 1.4.7 Bug

### ISSUE THREAD

**Issue by Deanobro** [atampy25/simple-mod-framework-installer#1]
Hangs when installing new mods. Screenshot of where it's stuck.
![SMF Hang](https://user-images.githubusercontent.com/99329384/235313086-12270d97-386f-40ee-82a1-5df9e87978ca.jpg)

**Comment by atampy25** [comment]
Wrong repository - this is for the installer EXE only; the framework is at https://github.com/atampy25/simple-mod-framework.

Try deleting the "staging" folder in the Mod Manager folder - it should resolve this issue.

**Comment by Deanobro** [comment]
Thanks for getting in touch.
 I use the EXE to install so thought I posted in the right place.
There is no staging folder in the mod manager folder (for me) and it was a
clean install as I usually use Nexus.
I grabbed the installer from GitHub due to the file flag on Nexus.
Extracting the Zips directly has sorted the issue for now so I'll reinstall
when it's been green lit on Nexus.
Thanks for your help,

D
